tests/TestFramework/Config/InfectionConfigTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+namespace Infection\Tests\TestFramework\Config;
```

Comment:
Incorrect namespace. There is already a test file for `InfectionConfig` class: `tests/Config/InfectionConfigTest.php`